R/class-forecast-nominal.R

```diff
@@ -0,0 +1,155 @@
+#' @export
+#' @keywords check-forecasts
+#' @importFrom checkmate assert_names assert_set_equal test_set_equal
+assert_forecast.forecast_nominal <- function(
```

Comment:
I would start with the as_forecast.<class> and define this later to avoid the reader being like what is this for

Reply:
yes makes sense! And I will take another look at it to make the order of functions somewhat consistent across files

Reply:
done